packages/cyberstorm/src/components/componentStyles/Pagination.module.css

```diff
@@ -0,0 +1,20 @@
+.root {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  align-items: center;
+  justify-content: center;
+
+  width: 407px;
+  height: 36px;
+  padding: 0;
+}
+
+.disabled {
+  pointer-events: none;
+}
+
+.disabled:hover {
+  background-color: transparent;
+  cursor: default;
+}
```

Comment:
These are redundant, as the rule `pointer-events:none` in the previous selector already render the elements unhoverable.

Reply:
Aight, deleted these so we don't have unnecessary things hanging 👍